src/main/java/io/leonis/subra/gui/field/GaussianPotentialFieldBackground.java

```diff
@@ -0,0 +1,25 @@
+package io.leonis.subra.gui.field;
+
+import com.googlecode.lanterna.TextColor;
+import io.leonis.subra.math.spatial.GaussianPotentialField;
+import java.util.function.*;
+import lombok.AllArgsConstructor;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+
+/**
+ * @author Jeroen de Jong
+ */
+@AllArgsConstructor
+public class GaussianPotentialFieldBackground implements BiFunction<Integer, Integer, TextColor> {
+
+  private final static double LIMIT = 100;
+  private final GaussianPotentialField potentialField;
+  private final Function<Double, TextColor.RGB> gradient;
+
+  @Override
+  public TextColor apply(final Integer x, final Integer y) {
+    final INDArray positionVector = potentialField.getPotential(Nd4j.create(new double[]{x, y}));
```

Comment:
should be called `potential`

Reply:
don't forget state vectors are column vectors. if you don't specify the dimensions of the array then nd4j will give you a row vector. try this:

`potentialField.getPotential(Nd4j.create(new double[]{x, y}, new int[]{2, 1}));`